python/py-elasticsearch/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-elasticsearch
-version             8.3.1
+version             8.9.0
 platforms           darwin
```

Comment:
it looks like that the port doesn't contain any compiled code and thus is independent of the architecture? Please verify, but if so one should likely set
```
platforms {darwin any}
supported_archs noarch
```

Reply:
Thanks, done.